### DISCUSSION TITLE
is Copilot not available with the Github enterprise for startups coupon ?

### DISCUSSION THREAD

**Discussion by hayad7** [community/community#121367] [Enterprise]
We are facing a billing issue with our GitHub for Startups Enterprise Plan. Our account was supposed to receive a $420 monthly discount for one year. However, our credit card was charged the full amount, causing a payment failure and account disablement. I suspect that this is due to our Copilot usage. I thought that the Coupon would cover all our Github costs.

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation, you need to verify it


Access the [Github Support](https://support.github.com/)

Click on My Ticket -> New Ticket -> Finally describe it

If I supported you consider to mark my answer. Good Luck!

**Reply by hayad7** [reply]
Thank you for the reply!
I already opened a ticket 13 hours ago, but got no reply.

**Comment by madhuatdabbl** [comment]
How did you redeem the coupon? We are not being able to redeem the coupon
